site/assets/scss/_boosted.scss

```diff
@@ -15,6 +15,11 @@ body {
   }
 }
 
+html,
+body {
+  min-height: 100%;
```

Comment:
It works best with `min-height: 100vh` here :

100% : 
![image](https://user-images.githubusercontent.com/91960143/160131138-46f008fd-0817-49b1-a15a-a1efc679515b.png)

100vh : 
![Uploading image.png…]()